modules/pensions/app/models/pensions/saved_claim.rb

```diff
@@ -71,10 +71,11 @@ def email
     ##
     # enqueue the sending of the submission confirmation email
     #
+    # @param id [Pensions::SavedClaim]
     # @see VANotify::EmailJob
     #
-    def send_confirmation_email
-      return if email.blank?
+    def send_confirmation_email(saved_claim_id)
+      return if email.blank? || find_notification(saved_claim_id).present?
```

Comment:
Should not need to pass the claim_id as an argument.  It is sending the email for _this_ model instance